src/main/scala/org/allenai/wikitables/WikiTablesUtil.scala

```diff
@@ -185,7 +185,7 @@ object WikiTablesUtil {
     // Vocab consists of all words that appear more than once in
     // the training data and in the name of any entity.
     val vocab = IndexedList.create(wordCounts.getKeysAboveCountThreshold(1.9))
-    vocab.addAll(IndexedList.create(entityCounts.getKeysAboveCountThreshold(0.0)))
+    // vocab.addAll(IndexedList.create(entityCounts.getKeysAboveCountThreshold(0.0)))
```

Comment:
If you're going to leave this in, but commented out, can you leave a note for why it's commented out, so that we don't wonder why it's there when we come back looking at this code later?